dashboard/static/css/home.css

```diff
@@ -0,0 +1,3 @@
+form#add-push-application input[type="submit"] {
+    display: block;
+}
```

Comment:
What if we have `main.css` and `home.css` source files to keep the code separate and organized, but then have it all compile into a single `styles.css` that's included on all pages?

At some point the bandwidth saved by downloading just the optimized CSS a page needs is overcome by the requests and bandwidth spent downloading a different (though optimized) CSS resource on so many pages.